app/controllers/access_lint/rails/errors_controller.rb

```diff
@@ -0,0 +1,57 @@
+module AccessLint
+  module Rails
+    class ErrorsController < ApplicationController
+      def create
+        warnings.each do |warning|
+          logger.tagged("AccessLint") do
+            logger.warn("\"#{url}\" - #{warning.message}")
+          end
+        end
+
+        head :ok
+      end
+
+      private
+
+      def warnings
+        violations_params.map do |violation|
+          Warning.new(violation)
+        end
+      end
+
+      def violations_params
+        params.require(:violations)
+      end
+
+      def url
+        URI.parse(params.require(:url)).path
+      end
+
+      def logger
+        @logger ||= ActiveSupport::TaggedLogging.new(::Rails.logger)
+      end
+
+      class Warning
```

Comment:
This feels more like something tied to the domain model than to the controller. What do you think of moving it to `app/models`?